apps/nova/src/components/TutorProfile/Ratings.tsx

```diff
@@ -19,15 +19,15 @@ const NoTutorRatings: React.FC<{ tutorName: string | null }> = ({
 }) => {
   const intl = useFormatMessage();
   return (
-    <div className="flex gap-[88px] items-center justify-center">
+    <div className="flex gap-[88px] tw-relative items-center justify-between h-[294px] w-full">
```

Comment:
<img width="868" alt="Screenshot 2024-12-24 at 8 39 58 AM" src="https://github.com/user-attachments/assets/f7da5ddc-77d2-4ecb-a5c5-c9f636d35ca4" />

The position absolute will cause this behaviour. I believe that their is a better solution. If the motivation behind your solution is that the image will push the content below it down. We can then use negative margin on the content below it to offset this effect.